homeassistant/components/opengarage/cover.py

```diff
@@ -42,6 +44,9 @@
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_NAME): cv.string,
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+        vol.Optional(CONF_PROTOCOL, default=DEFAULT_PROTOCOL): vol.In(
```

Comment:
Maybe `CONF_SSL`? HTTP and HTTPS are the same protocol but a different transport layer.

Reply:
Fair point. I'll push a change. Thanks